develop/application-devs/system-parachains/interact-asset-hub.md

```diff
@@ -0,0 +1,230 @@
+  and deposits for proxy/multisig operations
+- Ability to pay transaction fees in certain assets. As in, accounts would **not** need DOT to exist
+  on-chain or pay fees
+
+The Asset Hub will use DOT as its native currency. Users can transfer DOT from the relay chain into
+the Asset Hub and use it natively. The relay chain will also accept DOT transfers from the Asset Hub
+back to the relay chain for staking, governance, or any other activity.
+
+Using the Asset Hub for DOT/KSM balance transfers will be much more efficient than the relay chain
+and is highly recommended. Until domain-specific parachains are built, the relay chain will still
+need to be used for staking and governance.
+
+## Assets Basics
+
+See the [assets pallet](https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame/assets){target=\_blank} for
+the most up-to-date info and reference documentation.
+
+Assets are stored as a map from an ID to information about the asset, including a management team,
+total supply, total number of accounts, its sufficiency for account existence, and more.
+Additionally, the asset owner can register metadata like the name, symbol, and number of decimals
+for representation.
+
+Some assets, as determined by on-chain governance, are regarded as “sufficient”. Sufficiency means
+that the asset balance is enough to create the account on-chain, with no need for the DOT/KSM
+existential deposit. Likewise, you cannot send a non-sufficient asset to an account that doesn't
+exist. Sufficient assets can be used to pay transaction fees, meaning there is no need to hold DOT/KSM
+on the account.
+
+Assets do have a minimum balance (set by the creator), and if an account drops below that balance,
+the dust is lost.
+
+### Asset Operations
+
+The assets pallet has an interface for dealing with assets. See the [Integration](#integration)
+section below for how to fetch information and construct transactions.
+
+The main functions you will probably interact with are `transfer` and `transfer_keep_alive`. These functions transfer some `amount` (balance) of an `AssetId` (a `u32`, not a contract address) to another account.
+
+The assets pallet also provides an `approve_transfer`, `cancel_approval`, and `transfer_approved` interface for non-custodial operations.
+
+Asset transfers will result in an `assets.transferred` event. 
+
+<!-- TODO: would be nice to link a page on how to properly track and index events eventually -->
+
+Note that you can use the same addresses (except [pure proxies](https://wiki.polkadot.network/docs/learn-proxies-pure){target=\_blank}) on the Asset Hub that
+you use on the relay chain. The SS58 encodings are the same; only the chain information (genesis hash, etc.) will change on transaction construction.
+
+#### Paying Transaction Fees in Another Asset
+
+Users in the Asset Hub can pay the fees of their transactions with assets other than DOT. The only
```

Comment:
Same here, maybe:

Users have the option to pay transaction fees using assets other than the relay chain's native token.